src/components/form/examples/form-with-help.tsx

```diff
@@ -0,0 +1,51 @@
+import { Component, h, State } from '@stencil/core';
+import { schema } from './help-form-schema';
+
+/**
+ * Form fields with help
+ * It's possible to add extensive help to any form element.
+ * The string you provide can be in Markdown format,
+ * empowering you present a rich-text experience to the user,
+ * including bullet points, hyperlinks, etc…
+ * Read more on [Help](#/component/limel-help) component.
+ *
+ * :::note
+ * This "help" is not the same as providing a so called "helper text".
+ * Helper text is a short description of the input fields
+ * that becomes visible when the user click on the fields.
+ * It is most often used to render validation errors, or provide
+ * brief clues about the field or its expected value.
```

Comment:
Validation errors is maybe not the best first example after explaining that helper text becomes visible when clicking the field. That's not the behaviour when displaying errors. I suggest shorten this sentence, to just mention that it's _also_ used for validation errors. We're already pointing the input field examples for more details.

```suggestion
 * It is also used to render validation errors when a form field has invalid input.
```

Reply:
> Validation errors is maybe not the best first example after explaining that helper text becomes visible when clicking the field.

That's not true. We force-show the helper text, when field is invalid.

Reply:
> That's not true. We force-show the helper text, when field is invalid.

That's what I meant. So it's inconsistent with the preceding definition of helper text as "a short description of the input fields that becomes visible when the user click on the fields."

My suggestion is that we present validation errors as an alternative use of helper text, that's an exception (always shown when field is invalid), rather than as the main use.